### PR TITLE
rename module to tuneme

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ install:
   - pip install -r requirements-dev.txt
   - pip install -e .
 script:
-  - flake8 unpfayep
-  - py.test --ds=project.settings --verbose --cov unpfayep unpfayep
+  - flake8 tuneme
```
Comment (smn) [inline .travis.yml:12]: This should be able to be just `flake8` without `tuneme` but that may be a scaffolding problem.

**Comment by smn** [comment]
:+1: from me

File: .travis.yml (hunk 1 above)
Comment (miltontony) [inline .travis.yml:12]: yeah, it's from the scaffold. I'll update it

File: .deploy.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-buildscript: unpfayep/sideloader/build.sh
-postinstall: unpfayep/sideloader/postinstall.sh
+buildscript: sideloader/build.sh
```
Comment (miltontony) [inline .deploy.yaml:1]: @smn this also needs to be updated in the scaffold (since you've removed the 2 levels deep problem)

**Pull request merged**